.github/workflows/ci.yml

```diff
@@ -1,48 +1,118 @@
 name: CI
-
-on:
+'on':
```

Comment:
Using `'on'` is a bit awkward. We can keep `on` here.

Reply:
OK